data/air_table.py

```diff
@@ -0,0 +1,75 @@
+from typing import Tuple
+
+from airtable import Airtable
+from collections import defaultdict
+from core import config as settings
+
+
+def check_valid_score(data: dict) -> None:
+    for key in data:
+        assert data[key] == 5, f"problem id : {key}\n 키워드, 유사도 점수의 합은 항상 각각 5점 이어야 합니다!!"
+
+
+def get_keyword() -> Tuple[dict, defaultdict]:
+    keyword_view = Airtable(settings.AIR_TABLE_APP_NAME, "keyword", settings.AIR_TABLE_API_KEY).get_all()
+    problem_keywords = defaultdict(list)
+    problem_keywords_score = defaultdict(int)
+    keywords = {}
+    for keyword in keyword_view:
+        if keyword['fields'] == {}:
```

Comment:
```suggestion
        if not keyword['fields']:
```

dictionary가 비어있는지 체크하는 것 같은데, 참고해주세요

https://stackoverflow.com/questions/23177439/python-checking-if-a-dictionary-is-empty-doesnt-seem-to-work

Reply:
@kshired 리뷰 보고 저게 더 가독성이 좋겠네? 싶어서 바꾸려다가 보내주신 참고링크를 보았는데
```python
a = {}
if a == {}:  # 1
    print ('empty dict')
if not a:  # 2
    print ('empty dict')
```
Although method 1st is more strict as when a = None, method 1 will provide correct result but method 2 will give an incorrect result.
라는 내용이 있어요!!
둘 다 같은 결과지만 {}으로 제한해주는게 조금 더 명확하게 의도를 나타내는거라는 생각이 들어요!

Reply:
@ekzm8523 keyword가 iterator에 의해 가져와지는 애들이라 `None` 일 가능성이 없는데, 어떻게 생각하나요!

추가로 아래 airtable의 코드에서도 None을 return하지 않기 때문에 not 을 사용해도 괜찮다고 생각합니다.

```python
    def get_all(self, **options):
        """
        Retrieves all records repetitively and returns a single list.

        >>> airtable.get_all()
        >>> airtable.get_all(view='MyView', fields=['ColA', '-ColB'])
        >>> airtable.get_all(maxRecords=50)
        [{'fields': ... }, ...]


    Keyword Args:
            max_records (``int``, optional): The maximum total number of
                records that will be returned. See :any:`MaxRecordsParam`
            view (``str``, optional): The name or ID of a view.
                See :any:`ViewParam`.
            fields (``str``, ``list``, optional): Name of field or fields to
                be retrieved. Default is all fields. See :any:`FieldsParam`.
            sort (``list``, optional): List of fields to sort by.
                Default order is ascending. See :any:`SortParam`.
            formula (``str``, optional): Airtable formula.
                See :any:`FormulaParam`.

        Returns:
            records (``list``): List of Records

        >>> records = get_all(maxRecords=3, view='All')

        """
        all_records = []
        for records in self.get_iter(**options):
            all_records.extend(records)
        return all_records
```

Reply:
참고2

## Pep-8 Recommended Style

```python
if py_list:
	print('List is not empty')
if not py_list:
	print('List empty')
```

For this, let's take a look at [Truth Value Testing](https://docs.python.org/3/library/stdtypes.html#truth-value-testing). The official docs state that:

Here are most of the built-in objects considered false:

constants defined to be false: None and False.
zero of any numeric type: 0, 0.0, 0j, Decimal(0), Fraction(0, 1)
empty sequences and collections: '', (), [], {}, set(), range(0)
As an empty list is in fact just a empty collection, it will be converted to a boolean value of False. Therefore, if py_list is empty, it will converted to False.

The second statement is pretty similar, except not will invert the a false condition to a true one. This approach is very similar to the if(len(list)) approach.

This is the preferred approach as it's the cleanest and shortest solution there is.

Reply:
@kshired  맞아요 결국 둘 다 같은 결과고 not을 써도 문제될게 없지만 굳이 더 넓은 범위의 not을 쓰는 것 보단 피하고자 하는 case를 명확하게 지정해주는게 더 효과적이라 생각해요 ( 원래는 별 생각 없었는데 주셨던 참고링크를 보며 든 생각.. ) not A 보다 A is None을 더 권장하는 것과 같은 이유입니다!